integration/src/main/java/org/mobilitydata/gtfsvalidator/integration/Main.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2021 MobilityData IO
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.mobilitydata.gtfsvalidator.integration;
+
+public class Main {
+  public static void main(String[] argv) {
+    org.mobilitydata.gtfsvalidator.cli.Main.main(argv);
```

Comment:
We don't actually need to run the validator here within this module.

We can instead do the following in the GitHub Action script:
1. Build the CLI project with the latest changes and run the gtfs-validator JAR in the GitHub action (i.e., what we're currently doing)
2. Download the latest release JAR from https://github.com/MobilityData/gtfs-validator/releases (GitHub might be able [to do this directly](https://github.com/marketplace/actions/fetch-github-release-asset) vs. making an HTTP request?) and run it

We'll need to make sure the two sets of output don't collide.

Then in this module we can loop through all output files from both validators, read them in, and compare the results (i.e., v2.0.0 had 3 errors for agency-x while this PR had 10 errors for agency-x, so something went wrong).

Reply:
Thanks @barbeau, will modify accordingly!